### PR TITLE
docs - rename embedding

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
This PR changes names for embedding products in the docs (and only in the docs).

- Full-app embedding -> Interactive embedding.
- Signed/standalone -> Static embedding.

### CODE REVIEW

File: docs/embedding/static-embedding.md [hunk 1]
```diff
@@ -1,22 +1,24 @@
 ---
-title: Signed embedding
+title: Static embedding
+redirect_from:
+- /docs/latest/embedding/static-embedding
```
Comment (jeff-bruemmer) [inline docs/embedding/static-embedding.md:4]:
```suggestion
- /docs/latest/embedding/signed-embedding
```

File: docs/embedding/static-embedding-parameters.md [hunk 2]
```diff
@@ -1,12 +1,14 @@
 ---
-title: Parameters for signed embeds
+title: Parameters for static embeds
+redirect_from:
+- /docs/latest/embedding/static-embedding-parameters
```
Comment (jeff-bruemmer) [inline docs/embedding/static-embedding-parameters.md:4]:
```suggestion
- /docs/latest/embedding/signed-embedding-parameters
```

File: docs/embedding/start.md [hunk 3]
```diff
@@ -8,15 +8,15 @@ title: Embedding overview
...
+## [Interactive embedding](./interactive-embedding.md)
 
-The solution to self-service customer analytics: embed the full Metabase app in your app. Full-app embedding integrates with your data permissions to let people slice and dice data on their own using Metabase's query builder.
+The solution to self-service customer analytics: embed the full Metabase app in your app. Interactive embedding integrates with your data permissions to let people slice and dice data on their own using Metabase's query builder.
 
-## [Signed embedding](./signed-embedding.md)
+## [Static embedding](./static-embedding.md)
 
-Also known as Standalone Embedding, Signed Embedding offers drill-through with custom destinations, so you can define what happens when people click on a chart, like sending people to another chart or URL–all while securing the underlying data.
+Also known as Static Embedding, Static embedding offers drill-through with custom destinations, so you can define what happens when people click on a chart, like sending people to another chart or URL–all while securing the underlying data.
```
Comment (albertoperdomo) [inline docs/embedding/start.md:17]: Seems we are repeating static, I think the first should be signed as the aka?

```suggestion
Also known as Signed Embedding, Static embedding offers drill-through with custom destinations, so you can define what happens when people click on a chart, like sending people to another chart or URL–all while securing the underlying data.
```

File: docs/embedding/introduction.md [hunk 4]
```diff
@@ -12,21 +12,21 @@ You can embed Metabase tables, charts, and dashboards—even Metabase's query bu
...
+Interactive embedding is the only kind of embedding that [integrates with SSO and data permissions](./interactive-embedding.md) to enable true self-service access to the underlying data.
 
-**When to use full-app embedding**: when you want to [offer multi-tenant, self-service analytics](https://www.metabase.com/blog/why-full-app-embedding). With full-app embedding, people can create their own questions, dashboards, models, and more, all in their own data sandbox.
+**When to use interactive embedding**: when you want to [offer multi-tenant, self-service analytics](https://www.metabase.com/blog/why-full-app-embedding). With interactive embedding, people can create their own questions, dashboards, models, and more, all in their own data sandbox.
 
-## Signed embedding
+## Static embedding
 
-Also known as standalone embedding, signed embedding is a secure way to embed charts and dashboards.
+Also known as static embedding, static embedding is a secure way to embed charts and dashboards.
```
Comment (albertoperdomo) [inline docs/embedding/introduction.md:27]:
```suggestion
Also known as signed embedding, static embedding is a secure way to embed charts and dashboards.
```

File: docs/configuring-metabase/fonts.md [hunk 5]
```diff
@@ -122,7 +122,7 @@ To support multiple character sets, for example both Latin and Cyrillic, you'll
 
 ## Customizing the font for individual embedded items
 
-In addition to the [included fonts](#included-fonts), if you set a custom font for your Metabase, that font will be selectable from "Use instance font" in [signed embeds](../embedding/signed-embedding.md).
+In addition to the [included fonts](#included-fonts), if you set a custom font for your Metabase, that font will be selectable from "Use instance font" in [signed embeds](../embedding/static-embedding.md).
```
Comment (albertoperdomo) [inline docs/configuring-metabase/fonts.md:125]:
```suggestion
In addition to the [included fonts](#included-fonts), if you set a custom font for your Metabase, that font will be selectable from "Use instance font" in [static embeds](../embedding/static-embedding.md).
```

**Review by albertoperdomo — CHANGES_REQUESTED** [review]
I think I spotted some places where we were still accidentally using the old naming. In some cases it could be perhaps intentional?

File: docs/embedding/static-embedding-parameters.md [hunk 6]
```diff
@@ -1,12 +1,14 @@
 ---
-title: Parameters for signed embeds
+title: Parameters for static embeds
+redirect_from:
+- /docs/latest/embedding/signed-embedding-parameters
 ---
 
 # Parameters for signed embeds
```
Comment (jeff-bruemmer) [inline docs/embedding/static-embedding-parameters.md:7]:
```suggestion
# Parameters for static embeds
```

File: docs/embedding/static-embedding-parameters.md [hunk 7]
```diff
@@ -1,12 +1,14 @@
...
 
-Also known as: parameters for standalone embeds.
+Also known as: parameters for static embeds.
```
Comment (jeff-bruemmer) [inline docs/embedding/static-embedding-parameters.md:9]:
```suggestion
Also known as: parameters for signed embeds, or standalone embeds.
```

File: docs/embedding/interactive-embedding.md [hunk 8]
```diff
@@ -98,13 +99,13 @@ If you want to embed Metabase in another domain (say, if Metabase is hosted at `
...
+If you set this environment variable to "None", you must use HTTPS in Metabase to prevent browsers from rejecting the request. For more information, see MDN's documentation on [SameSite cookies](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite).
 
 Note that `SameSite=None` is incompatible with most Safari and iOS browser versions (including any browser that runs on iOS, such as Chrome on iOS).
 
-## Securing full-app embeds
+## Securing interactive embeds
 
-Metabase uses HTTP cookies to authenticate people and keep them signed into your embedded Metabase, even when someone closes their browser session. If you enjoy diagrammed auth flows, check out [Full-app embedding with SSO](https://www.metabase.com/learn/customer-facing-analytics/securing-embeds#full-app-embedding-with-sso).
+Metabase uses HTTP cookies to authenticate people and keep them signed into your embedded Metabase, even when someone closes their browser session. If you enjoy diagrammed auth flows, check out [Interactive embedding with SSO](https://www.metabase.com/learn/customer-facing-analytics/securing-embeds).
```
Comment (albertoperdomo) [inline docs/embedding/interactive-embedding.md:108]: Why remove the anchor link?

File: docs/paid-features/overview.md [hunk 9]
```diff
@@ -34,7 +34,7 @@ Paid plans include more ways to manage permissions, including data sandboxing, w
 - [Download permissions](../permissions/data.md#download-results)
 - [Connection impersonation](../permissions/data.md#impersonation-access)
 - [Database management permissions](../permissions/data.md#manage-database)
-- [Data model management permissions](../permissions/data.md#manage-data-model)
+- [Table metadata management permissions](../permissions/data.md#manage-table-metadata)
```
Comment (albertoperdomo) [inline docs/paid-features/overview.md:37]: This seems unrelated to the naming change, but I guess it makes sense? I don't have the context so I don't know.

**Review by albertoperdomo — APPROVED** [review]
Left a couple of comments, but not sure they are relevant or could be ignored.

File: docs/embedding/interactive-embedding.md (hunk 8 above)
Comment (jeff-bruemmer) [reply docs/embedding/interactive-embedding.md:108]: We'll eventually change the destination link, so removing it here now.

File: docs/paid-features/overview.md (hunk 9 above)
Comment (jeff-bruemmer) [reply docs/paid-features/overview.md:37]: We renamed the admin "Data model" tab to "Table metadata" to distinguish it from models. Just say this link issue when reviewing this PR, so I tacked the change on here while I was at it.

**Pull request merged**